app/src/main/java/com/xzyht/notifyrelay/sync/ProtocolTransports.kt

```
@@ -24,12 +24,85 @@ import java.net.Socket
+            val localIp = getLocalIpAddress()
+            val deviceType = "android"
+
+            val req = "PAIRING_INIT:${manager.uuid}:$tmpPublicKey:$localIp:$batteryStr:$deviceType\n"
+            writer.write(req)
+            writer.flush()
+            val resp = reader.readLine()
+            try { writer.close() } catch (_: Exception) {}
+            try { reader.close() } catch (_: Exception) {}
+            try { socket.close() } catch (_: Exception) {}
+            resp
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)